pyinfra/api/inventories/kubernetes.py

```diff
@@ -0,0 +1,26 @@
+from pyinfra.api.connectors.util import run_local_process, split_combined_output
+from pyinfra.api.exceptions import InventoryError
+
+
+def get_pods(selector, namespace='default', all_namespaces=False, container=None):
+    if all_namespaces:
+        ns = ['-A']
+    else:
+        ns = ['-n', namespace]
+
+    return_code, combined_output = run_local_process(
+        ['"$@"', '-',
+         'kubectl', 'get', 'pods', *ns, '-l', selector, '--template',
```

Comment:
Unfortunately Python 2 doesn't support the `*ns` expansion here - you'll need to create a list of arguments, append the namespace ones and then the rest before passing to the `run_local_process` function.